server/implementation/src/main/java/io/smallrye/graphql/spi/config/Config.java

```diff
@@ -86,8 +86,13 @@ default boolean isListed(Class throwableClass, List<String> classNames) {
         }
 
         // Check that specific class
-        if (classNames.contains(throwableClass.getName())) {
-            return true;
+        for (String configuredValue : classNames) {
+            if (configuredValue.equals(throwableClass.getName())) {
+                return true;
+            } else if (configuredValue.endsWith("*")) {
+                String values = configuredValue.substring(0, configuredValue.length() - 2);
+                return throwableClass.getName().startsWith(values);
```

Comment:
Here only return if true. If false, we still want to iterate the rest of the for loop

Reply:
Thanks, you are right. We need to iterate the loop if it is false.